src/common/dateUtils.ts

```diff
@@ -0,0 +1,17 @@
+import { padStart } from "lodash";
```

Comment:
Is there a reason between using the lodash `padStart` vs the [native string method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart) ?